spec/weather.spec.js

```diff
@@ -0,0 +1,34 @@
+const { assertEquals } = require(`../test-framework`);
+const Weather = require("../src/weather");
+
+let weather;
+let weatherArray = [];
+let whatsTheWeather;
+let expected;
+let actual;
+let result;
+
+// Test W1
+console.log(`-----------------------------------------------------------------------------------------------------------------------`);
+console.log(`Test W1 - Weather Testing - When calling getWeather, a random weather condition should be called`);
+
+// Arrange
+weather = new Weather;
+weatherArray = [`clear`, `stormy`];
+expected = true
+
+// Act
+const checkWeather = () => weather.getWeather();
```

Comment:
Was unsure if you are allowed to create functions in the spec file.
I believe there is a more efficient, and cleaner, way of doing this